### PR TITLE
E2E tests for item closing

### LINKED ISSUE #623: e2e: Write tests for closing projects/subprojects/workflowitems
Write e2e tests for closing  projects/subprojects/workflowitems where only the assignee is allowed to close it

### Solution 
Create extra close.spec files for project/subproject

### TODO

- [x] Create project close e2e-tests
- [x] Create subproject close e2e-tests
- [x] Check workflowitem close e2e-tests

### PR DESCRIPTION
**Pull request by m-janos**
### Checklist

- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/openkfw/TruBudget/blob/master/CONTRIBUTING.md).

### Description

* E2E tests for closing subproject were added 
* E2E tests for closing a project and workflow item already there

Closes #623

### CODE REVIEW

File: e2e-test/cypress/integration/subproject_close_spec.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+  before(() => {
+    baseUrl = Cypress.env("API_BASE_URL") || `${Cypress.config("baseUrl")}/test`;
+    apiRoute = baseUrl.toLowerCase().includes("test") ? "/test/api" : "/api";
+    cy.login();
+    cy.createProject("sp-close", "Project for test of suproject closing")
+      .then(({ id }) => {
+        projectId = id;
+      })
+      .then(() => {
```
Comment (daniel-arnauer) [inline e2e-test/cypress/integration/subproject_close_spec.js:14]: remove these2 Lines

**Comment by daniel-arnauer** [comment]
Remove the merge commit and do a rebase instead

File: e2e-test/cypress/integration/subproject_close_spec.js (hunk 1 above)
Comment (m-janos) [reply e2e-test/cypress/integration/subproject_close_spec.js]: Thanks, looks better now.

**Review by daniel-arnauer — APPROVED** [review]
approved

**Pull request merged**